### ISSUE TITLE
Propose method and client-level specification of desired units for input/output

### ISSUE THREAD

**Issue by bdpedigo** [seung-lab/CAVEclient#150]
we discussed two interfaces for specifying the units for dimensions for inputs and outputs:

1. methods could implement a `units`/`resolution`/etc. flag which denotes what units a user is inputting (e.g. for a bounding box) or expecting (e.g. for a spatial location return value). the default would be `None` which would not change the current behavior
2. the CAVEClient class could also implement the same flag, and would also be `None` by default. specifying this would lead to all subsequent calls being in those units, unless overridden at the method level

**Comment by bdpedigo** [comment]
Functions/methods with some kind of spatial input/return:

- most things in annotation, e.g.
  - `annotation.create_table`: `voxel_resolution`
     - already has a flag for this kind of thing
  - `annotation.post_annotation_df`: `position_columns` ? 
  - `annotation.update_annotation_df`
- chunkedgraph
  - `do_merge`
  - `execute_split`
  - `find_path`
  - `get_contact_sites`
  - `get_operation_details`
  - `get_subgraph`
  - `preview_split`
- l2cache
  - `get_data`
    - at least for the nm properties... not sure what to do here  
- materialization
  - basically anything with query in the name
